src/toffy/fov_watcher.py

```diff
@@ -540,12 +541,38 @@ def start_watcher(
         intermediate_callback (Callable[[None], None]):
             function defined as run callback overriden as fov callback. assemble this using
             `watcher_callbacks.build_callbacks`
+        run_folder_timeout (int):
+            how long to wait for the run folder to appear before timing out, in seconds.
+            note that the watcher cannot begin until this run folder appears.
         completion_check_time (int):
             how long to wait before checking watcher completion, in seconds.
             note, this doesn't effect the watcher itself, just when this wrapper function exits.
         zero_size_timeout (int):
             number of seconds to wait for non-zero file size
     """
+    # warn the user to explicitly confirm the run folder
+    if not os.path.exists(run_folder):
+        warnings.warn(
+            f"Waiting for {run_folder}. Please first double check that your run data "
+            "doesn't already exist under a slightly different name in D:\\Data. "
+            "Sometimes, the CACs change capitalization or add extra characters to the run folder. "
+            "If this happens, stop the watcher and update the run_name variable in the notebook "
+            "before trying again."
+        )
+
+    # allow the watcher to
```

Comment:
The suspense! what is it allowed to do? 🤔

Reply:
I'm giving the developer an opportunity to use their imagination 🤣